src/clib/pioc_support.c

```diff
@@ -167,6 +239,36 @@ void pio_finalize_logging(void)
 #endif /* PIO_ENABLE_LOGGING */
 }
 
+/**
```

Comment:
I'm confused. You introduced your own timer, but are still making changes to gptl code? Are you also using GPTL?